library/src/main/scala/scala/scalajs/js/Iterable.scala

```diff
@@ -0,0 +1,23 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js API               **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013-2015, LAMP/EPFL   **
```

Comment:
2017.

Seems we should have some global pass over these header comments, one of these days.